terraform/trial_rg/services.tf

```diff
@@ -12,18 +22,29 @@ module "trial_app_service_site" {
   docker_image_tag    = var.site_image_tag
 
   settings = {
-    "WEBSITES_PORT"               = "8080" # The container is listening on 8080
+    "SPRING_APPLICATION_NAME"                 = "site-service"
```

Comment:
You're overriding the default application name that's packaged in the source. Is it required?

Reply:
no